packages/cyberstorm/src/components/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -3,41 +3,133 @@ import React, { PropsWithChildren } from "react";
 import { faHouse } from "@fortawesome/pro-regular-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import styles from "./BreadCrumbs.module.css";
-import { IndexLink } from "../Links/Links";
+import {
+  CommunitiesLink,
+  CommunityLink,
+  IndexLink,
+  PackageLink,
+  TeamsLink,
+} from "../Links/Links";
+
+type PageTitleCrumbProps = PropsWithChildren<{
+  pageTitle: string;
+}>;
 
 type BreadCrumbsProps = PropsWithChildren<{
-  excludeHome?: boolean;
+  pageTitle: string;
 }>;
 
-export function BreadCrumbs(props: BreadCrumbsProps) {
-  const children = React.Children.toArray(props.children);
-  if (!props.excludeHome) {
-    children.unshift(<DefaultHomeCrumb />);
-  }
-
-  const nodes = children.map((node, index) => {
-    const isLast = index == children.length - 1;
-    return (
-      <div
-        key={index}
-        className={`${index == 0 ? styles.outer__start : ""} ${styles.outer} ${
-          isLast ? styles.outer__end : ""
-        }`}
-      >
-        <div className={`${styles.inner} ${isLast ? styles.inner__end : ""}`}>
-          {node}
-        </div>
-      </div>
-    );
-  });
+interface PackageBreadCrumbsProps extends BreadCrumbsProps {
+  community: string;
+}
 
-  return <div className={styles.root}>{nodes}</div>;
+interface PackageDependantsBreadCrumbsProps {
```

Comment:
If we need these interfaces, I think we can do with some reusability here. E.g. this could be just `PackageBreadCrumbsProps` since the three props will be shared by many package related views.

But I hope we don't need to drop the BreadCrumbs component and these interfaces won't be needed.